app/src/main/res/values/strings.xml

```diff
@@ -609,5 +609,8 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_address_answer_no_housenumber_message2">Is this correct and is it not just part of a building?</string>
 
     <string name="notification_channel_download">"Download"</string>
+    <string name="quest_segregated_title">How footway and cycleway are combined here?</string>
```

Comment:
Hmm, my first suggestion is:

How are the foot- and cycleway laid out here?

1. Segregated from one another
2. Cyclists and pedestrians share the space

Reply:
Thanks, I really like this version.